data/mods/DinoMod/monsters/zed-dinosaur.json

```
@@ -835,5 +835,25 @@
     "armor_bullet": 1,
     "description": "A small four-legged plant-eating dinosaur juvenile corpse with natural bone armor plates.",
     "upgrades": { "half_life": 14, "into": "mon_zalamosaurus" }
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_zianzhousaurus",
+    "name": { "str": "Qianz zombie" },
+    "looks_like": "mon_zyrannosaurus",
+    "volume": "550 L",
+    "weight": "770 kg",
+    "speed": 175,
```

```suggestion
    "speed": 60,
```
Zombies are a little slower, about 80% speed